.github/workflows/test.yml

```diff
@@ -238,7 +238,7 @@ jobs:
           save-if: ${{ !startsWith(github.ref, 'refs/heads/gh-readonly-queue') }}
           workspaces: |
             ./rust/main
-            ${{ matrix.e2e-type == 'non-cosmwasm' && './rust/sealevel' || '' }}
+            ${{ matrix.e2e-type != 'cosmwasm' && './rust/sealevel' || '' }}
```

Comment:
i think this can be `matrix.e2e-type = 'sealevel'` now, since that's the only place where the `'./rust/sealevel'` is needed